docs/arvo/subject.md

```diff
@@ -58,8 +58,8 @@ Hoon, we call the code executed the "formula" and its context the
       (add b $(b (add b 5)))    ::
                                 ::
     ++  sum                     ::
-      |=  a/@u                  ::
-      (add (five a) (three a))  ::
+      |=  {a/@u b/@u}           ::
+      (add (five a) (three b))  ::
```

Comment:
This is the solution to the previous exercise; not sure if it should be included here or not, but `sum` is called with two atoms...